tests/test-sample.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+
+script_dir=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+nmsgtool_test=$script_dir/../src/nmsgtool
+infile=$script_dir/private-tests/202_sample.nmsg
+outfile=/tmp/testsample.json
+entry_count=5
```

Comment:
Could use a quick comment explaining how we got this (i.e. 1 out of every 3 x 15 = 5)

Reply:
Fixed!  Now the sample filter parameter is calculated from entries in the file and desired output entries.